pkg/datapath/iptables/iptables.go

```diff
@@ -233,6 +250,179 @@ func RemoveRules() {
 	}
 }
 
+func ingressProxyRule(cmd, destMatch, l4Match, markMatch, mark, port, name string) []string {
+	ret := []string{
+		"-t", "mangle",
+		cmd, ciliumPreMangleChain,
+		"-d", destMatch,
+		"-p", l4Match,
+		"-m", "mark", "--mark", markMatch,
+		"-m", "comment", "--comment", "cilium: TPROXY to host " + name + " proxy",
+		"-j", "TPROXY",
```

Comment:
This is a new requirement that we need to document in the system requirements. How common is TPROXY support in distribution configurations?

Reply:
Both TPROXY and NOTRACK are widely supported (#7435)